GUI/src/cz/fidentis/gui/comparison_batch/BatchRegistrationConfiguration.java

```diff
@@ -1277,74 +1291,125 @@ private void jComboBox2ActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FI
                 discPanel.setVisible(false);
                 break;
         }
-    }//GEN-LAST:event_jComboBox2ActionPerformed
+        
+        getContext().setMethod(undersamplingComboBox.getSelectedIndex());
+    }//GEN-LAST:event_undersamplingComboBoxActionPerformed
 
-    private void jRadioButton1ActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_jRadioButton1ActionPerformed
+    private void percentageRadioButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_percentageRadioButtonActionPerformed
         percentageSpinner.setEnabled(true);
         numberSpinner.setEnabled(false);
-    }//GEN-LAST:event_jRadioButton1ActionPerformed
+        getContext().setType(0);
+    }//GEN-LAST:event_percentageRadioButtonActionPerformed
 
-    private void jRadioButton2ActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_jRadioButton2ActionPerformed
+    private void numberRadioButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_numberRadioButtonActionPerformed
         percentageSpinner.setEnabled(false);
         numberSpinner.setEnabled(true);
-    }//GEN-LAST:event_jRadioButton2ActionPerformed
+        getContext().setType(1);
+    }//GEN-LAST:event_numberRadioButtonActionPerformed
 
     private void facesComboBoxActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_facesComboBoxActionPerformed
-
+        getContext().setTemplateIndex(facesComboBox.getSelectedIndex());
     }//GEN-LAST:event_facesComboBoxActionPerformed
 
     private void icpMetricComboBoxActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_icpMetricComboBoxActionPerformed
-        // TODO add your handling code here:
+        getContext().setIcpMetric((ICPmetric) icpMetricComboBox.getSelectedItem());
     }//GEN-LAST:event_icpMetricComboBoxActionPerformed
 
     private void symModelsCheckboxActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_symModelsCheckboxActionPerformed
-        // TODO add your handling code here:
+        getContext().setUseSymmetry(symModelsCheckbox.isSelected());
     }//GEN-LAST:event_symModelsCheckboxActionPerformed
+
+    private void facesComboBoxItemStateChanged(java.awt.event.ItemEvent evt) {//GEN-FIRST:event_facesComboBoxItemStateChanged
+        // TODO add your handling code here:
```

Comment:
tu asi patri nejaky kod alebo je to mozno nejaky pozostatok z toho mergeovania co si spominala?